CHANGES

```diff
@@ -1,6 +1,16 @@
 Babel Changelog
 ===============
 
+Version 2.9.2
+--------------
```

Comment:
I appreciate the changelog entry – I usually distill them by hand around release time, so this spares me some effort.

However, this'll probably end up in 2.10, so maybe just

```suggestion
Next version
------------
```
for now?